week3/intro.txt

```diff
@@ -1,6 +1,6 @@
 My five favorite things are:
-- 
-- 
-- 
-- 
-- 
+- Sport(Volleyball)
```

Comment:
I think volleyball is a pretty cool sport to play too!

Reply:
I couldn't agree more. I even started a coed volleyball club at cc this quarter.